poethepoet/executor/base.py

```diff
@@ -151,6 +151,10 @@ def _execute_cmd(
         process. Using exec supports fewer options, and doesn't work on windows.
         """
 
+        if self.working_dir and not self.working_dir.is_dir():
```

Comment:
what do you think about moving this to the except block below? that way there's not extra logic on the happy path.

Reply:
> what do you think about moving this to the except block below? 

 I have no problem with it.